### PR TITLE
transport: fix get_request in linux

### PR DESCRIPTION
**Pull request by killagu**
mio default add EPOLLET to event flags, so epoll will use
edge-triggered mode. It may let poll miss some event, so
manually register the fd with only EPOLLIN flag to
use level-triggered mode.

### CODE REVIEW

File: src/transport/fusedev/linux_session.rs [hunk 1]
```diff
@@ -21,6 +21,7 @@ use nix::errno::Errno;
 use nix::fcntl::{fcntl, FcntlArg, OFlag};
 use nix::mount::{mount, umount2, MntFlags, MsFlags};
 use nix::poll::{poll, PollFd, PollFlags};
+use nix::sys::epoll::{epoll_ctl, EpollOp, EpollEvent, EpollFlags};
```
Comment (jiangliu) [inline src/transport/fusedev/linux_session.rs:24]: Is `nix::sys::epoll` an wrapper for `kqueue`?

Comment (killagu) [reply src/transport/fusedev/linux_session.rs:24]: No, it's epoll only.

**Comment by killagu** [comment]
With [FuseChannel#get_requests](https://github.com/cloud-hypervisor/fuse-backend-rs/blob/master/src/transport/fusedev/linux_session.rs#L223) and [Server#handle_message](https://github.com/cloud-hypervisor/fuse-backend-rs/blob/master/src/api/server/sync_io.rs#L40), we can only handle one message at the same loop. So we dependent the level-trigger mode.

**Comment by jiangliu** [comment]
> Yes, let's fix it with level trigger for now and do a refactor later to switch to edge trigger. wdyt? @jiangliu @liubogithub

I'm OK once we got the clear idea why it doesn't work with edge triggered signal.

**Pull request merged**